integration/bin/run-integration.sh

```diff
@@ -1,20 +1,48 @@
 #!/usr/bin/env bash
 
-# Usage: run-integration.sh COOK_PORT_1 COOK_PORT_2 COOK_SLAVE_PORT
-
+# Usage: run-integration.sh [OPTIONS] ARGS...
+#
 # Runs the integration tests for cook scheduler in a docker container.
-# COOK_PORT_1 (Optional) - Port for a cook instance running in docker.
-# COOK_PORT_2 (Optional) - If COOK_MULTI_CLUSTER is set, port for a second cook instance
-#                          running in docker.
-# COOK_SLAVE_PORT (Optional) - If COOK_MASTER_SLAVE is set, port for a cook slave instance
-#                              running in docker.
+#
+# Options:
+#   --port NUMBER        Port for a cook instance running in docker.
+#   --port-2 NUMBER      If COOK_MULTI_CLUSTER is set, port for a second cook instance running in docker.
+#   --slave-port NUMBER  If COOK_MASTER_SLAVE is set, port for a cook slave instance running in docker.
+#   --working-copy       Mount working directory as a docker volume for faster development feedback loop.
```

Comment:
Can this be changed to an opt-out flag? Maybe just `--no-mount`?

Reply:
Will do. I think you're the only one who regularly uses this script, so it's probably best to have it match your workflow.